ci/ray_ci/tester.py

```diff
@@ -182,7 +182,7 @@ def main(
         get_flaky_tests=run_flaky_tests,
     )
     success = container.run_tests(test_targets, test_arg)
-    sys.exit(0 if success else 1)
+    sys.exit(0 if success else 40)
```

Comment:
why 40? why not 42?

Reply:
42 sounds good ;) https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life.2C_the_Universe.2C_and_Everything_.2842.29